solutions/swb-reference/integration-tests/tests/multiStep/paab.test.ts

```diff
@@ -58,10 +58,11 @@ describe('multiStep environment test', () => {
   let project1Id: string;
   let project2Id: string;
   let project3Id: string;
+  let project4Id: string;
```

Comment:
Rather than creating a 4th project, can we run the test against PA2 not seeing project3?